examples/hello_world/client/android/cpp/client.cc

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright 2019 The Project Oak Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <android/log.h>
+#include <jni.h>
+#include <string>
+
+#include "examples/hello_world/proto/hello_world.grpc.pb.h"
+#include "examples/hello_world/proto/hello_world.pb.h"
+#include "examples/utils/utils.h"
+#include "include/grpcpp/grpcpp.h"
+#include "oak/client/application_client.h"
+#include "oak/client/manager_client.h"
+
+extern "C" {
+
+#define APPNAME "Oak JNI"
+#define JNI_LOG(MSG) __android_log_print(ANDROID_LOG_INFO, APPNAME, MSG);
+
+using ::oak::examples::hello_world::HelloRequest;
+using ::oak::examples::hello_world::HelloResponse;
+using ::oak::examples::hello_world::HelloWorld;
+
+// Global channel storage.
+uint CHANNEL_COUNT = 1;
+std::unordered_map<jint, std::unique_ptr<HelloWorld::Stub>> CHANNEL_MAP;
```

Comment:
Is this the coding standard for global variables? (uppercase?)

Reply:
Done